Doc upgrade guide (#6231)
cyberbotics/webots: docs/guide/upgrading-webots.md

# Installing Webots

> This chapter explains how to upgrade worlds and PROTO nodes from an older version of Webots.

Reviewer: ```suggestion
This chapter explains how to upgrade your simulations from an older version of Webots.
```

> If you have a world/PROTO which is not from the previous version but from an earlier one, you need to follow each upgrade instructions in order. For example if you have a world from R2021b, you first need to upgrade it to R2022a, then R2022b, and so on.

Reviewer: ```suggestion
If you have a Webots simulation which is not from the previous version but from an earlier one, you need to follow each upgrade instructions in order. For example if you have a world from R2021b, you first need to upgrade it to R2022a, then R2022b, and so on.
```

> *Note*: if you have a world/proto which is not from the previous version but from an earlier one, you need to follow each upgrade instructions in order. For example if you have a world from R2021b, you first need to upgrade it to R2022a, then R2022b, and so on.

Reviewer 2: Personally I would remove the note and write it as normal paragraph (and write each sentence on a new line).
Because there are more notes and description in this page.

Otherwise you should use the node formatting (starting with ``> **Note**:``)

> *Note*: some versions of Webots do not have a page in this section. It means that the compatibility was not broken by this version. I you want to know what new features this version introduces, please refers to the [blog](../blog/index.md) or the [changelog](../reference/changelog.md).

Reviewer 2: Personally I would remove the note and write it as normal paragraph (and write each sentence on a new line).
Because there are more notes and description in this page.

Otherwise you should use the node formatting (starting with ``> **Note**:``)
